lib/Makefile

```diff
@@ -71,3 +76,13 @@ subclean:
 	rm -f nbtsort.c
 
 LDFLAGS+=-Wl,--build-id
+
+pg_bulkload--$(BULKLOAD_VERSION).sql: pg_bulkload.sql.in
+	sed 's,MODULE_PATHNAME,$$libdir/$(MODULE_big),g' $< \
+	| sed 's,BULKLOAD_VERSION,$(BULKLOAD_VERSION),g' > $@
+
+pg_bulkload--unpackaged--$(BULKLOAD_VERSION).sql: pg_bulkload--unpackaged.sql.in
```

Comment:
Do we really need it now? 

"unpackaged" seems to be used for extension updates, but now pg_bulkload doesn't support it.
https://www.postgresql.org/docs/12/extend-extensions.html

Reply:
Additionally, is it better to add documentation how to upgrade pg_bulkload? Users need to drop extension and create extension.

Reply:
Yes, you are right.
pg_bulkload don't support ALTER EXTENSION to update the version.
So, I removed the file, and added description how to upgrade pg_bulkload.